binding.cc

```diff
@@ -1,62 +1,65 @@
-#include <node.h>
-#include <v8.h>
+#include <nan.h>
 
 #include <bcm_host.h>
 
-using namespace v8;
-
 static VCHI_INSTANCE_T vchi;
 static VCHI_CONNECTION_T *vchi_connections;
 
 static char buf[1024];
 
-Handle<Value> Request(const Arguments& args) {
-    HandleScope scope;
+NAN_METHOD(Request) {
+    if (info.Length() < 1) {
+        Nan::ThrowTypeError("Wrong number of arguments");
```

Comment:
As far as I am concerned, while `request()` is internal, here must be `assert()`